qa/tasks/mgr/test_orchestrator_cli.py

```diff
@@ -155,7 +155,6 @@ def test_host_rm(self):
         self._orch_cmd("host", "rm", "hostname")
 
     def test_mon_update(self):
-        self._orch_cmd("mon", "update", "3")
```

Comment:
interesting. this must work for rook. That's a bug in the orchestrator_cli. The code in _update_mons must not require and placement specs.

```suggestion
        # TODO: This test needs to be fixed:
        # self._orch_cmd("mon", "update", "3")
```

Reply:
@sebastian-philipp could you file a tracker ticket on this? i think a TODO line is probably not the best way to track the pending tasks.

Reply:
> @sebastian-philipp could you file a tracker ticket on this? i think a TODO line is probably not the best way to track the pending tasks.

#32499